src/amber/cli/commands/deploy.cr

```diff
@@ -147,15 +148,23 @@ module Amber::CLI
         remote_cmd "docker exec -id amberweb ./#{project_name}"
       end
 
-      def stop_and_remove
+      private def stop_and_remove
         cmds = ["docker rm -rf amberweb"]
         cmds << "docker rm -rf amberdb"
         remote_cmd(%Q(bash -c "#{cmds.join(" && ")}"))
       end
 
-      def update_project
+      private def update_project
         remote_cmd(%Q("cd amberproject && git pull"))
       end
+
+      private def server_name
+        @server_name.colorize(:green)
+      end
+
+      private def puts(msg)
```

Comment:
I don't think this should be defined for every command. Could you make it a protected method `::CLI::Command`